routes/v1.js

```diff
@@ -614,8 +636,11 @@ api.declare({
   name:       'rerunTask',
   scopes:     [
     [
+      // Legacy scopes
       'queue:rerun-task',
       'assume:scheduler-id:<schedulerId>/<taskGroupId>'
+    ], [
+      'queue:rerun-task:<schedulerId>/<taskGroupId>',
```

Comment:
Better to use `queue:rerun-task:<schedulerId>/<taskGroupId>/<taskId>`. Note, `queue:rerun-task:<schedulerId>/<taskGroupId>/*` would provide existing semantics, but allow finer grained access control if required, when a component should only be allowed to rerun an individual task in a task graph, rather than an arbitrary one.